content/rainstorms/8_conclusion.md

```diff
@@ -0,0 +1,9 @@
+---
+image: rainstorms/conclusion.png
+headline: Conclusion
+---
+
+## Conclusion
+
+This study uses a novel method to examine changes to the ingredients of heavy precipitation. Our results suggest that storms will have higher peak intensity, longer duration and will be more frequent across the whole of Europe. Current storms already produce a large number of flash floods, with their potential impact depending on land use, terrain slope, drainage, and other factors. SEPP increases would significantly increase this flash flood potential, as an MCS would be more likely to “stagnate” on a locality, exposing it to extreme precipitation of longer duration. Additionally, storm system movement is associated with upper level winds and, hence, large-scale dynamics. This may increase fluvial flood risk through consecutive events in one favorable synoptic setting spanning days or more; blocking is also favored by large-scale meandering patterns associated with slower flows. Understanding the underlying ingredients for heavy precipitation change is crucial from an impacts perspective, helping to discriminate controlling factors, which have wider applicability beyond those for a single accumulation period, and to identify the reliability of projected changes. This suggests that future studies should focus on precipitation accumulations over space and time.
```

Comment:
Maybe use some paragraphs here?

Reply:
Good point. The content is now split into three paragraphs.